topics/proteomics/tutorials/Proteogenomics/Proteogenomics_DBsearch/Protegenomics_RNAseqDBworkflow.yaml

```diff
@@ -0,0 +1,47 @@
+install_tool_dependencies: True
```

Comment:
can we have the folder name `Proteogenomics` be with small letters, like `proteogenomics`.

Reply:
I dont know how to edit it and write it in small letters. Whenever I do something like that, it creates a new folder. Do you have any suggestions on how to rename/edit it??

Reply:
`git mv topics/proteomics/tutorials/Proteogenomics topics/proteomics/tutorials/proteogenomics` should work afaik and then commit the change